docs/cedarling/cedarling-quick-start.md

```diff
@@ -10,14 +10,185 @@ tags:
 
 # Cedarling Quick Start Guide
 
-## This content is in progress
+## Introduction
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+In this guide, we'll demonstrate how to use [Jans Tarp](../../demos/jans-tarp) to register a client on an authorization server, and use the embedded cedarling webassembly app to reach an authorization decision. This will be done in three steps:
 
-## Have questions in the meantime?
+1. [Setting up a policy store](#policy-store-setup)
```

Comment:
I think "policy store" should be capitalized, because it's a "Cedarling Policy Store", not just any policy store. It would be more clear to say "Cedarling Policy Store" first, so it's clear.